### DISCUSSION TITLE
ScriptReference/ScriptableObject

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#59] [Notes]
# ScriptReference/ScriptableObject

https://docs.unity3d.com/ScriptReference/ScriptableObject.html

**Comment by vertxxyz** [comment]
ScriptableObjects used at runtime must be referenced into the build.
This can be achieved by direct references in the scene, the Resources folder, Asset Bundles/Addressables, and Player Settings' [preloaded assets](https://docs.unity3d.com/ScriptReference/PlayerSettings.SetPreloadedAssets.html).
